### ISSUE TITLE
Entity emissive textures don't work with sodium

### ISSUE THREAD

**Issue by VindSole** [MoreMcmeta/emissive-plugin#23]
**Describe the bug**
Entities emissive overlay texture don't seem to render or are not emissive when using sodium and Indium, also tested with embeddium.

**To reproduce**
1. Use Sodium/Indium
2. Install a resource pack with emissive texture overlays on entities
3. Entity emissive is not bright/doesn't work
4. Disable Sodium/Indium
5. Entity emissive looks as intended

**Expected behavior**
The entity's emissive texture should render/be brighter

**Screenshots or videos**
![image](https://github.com/MoreMcmeta/emissive-plugin/assets/63668266/1d2d4e60-5ecc-4a03-aac2-0e3860f0b201)

**Which Minecraft versions does this bug affect?**
1.20.6

**Mod loader**
Which mod loaders does this bug affect?

- [ V ] Fabric

**Mods list**
Sodium
Polytone
Entity Texture Features
Entity Model Features

**Resource pack**
[drowned_test.zip](https://github.com/MoreMcmeta/emissive-plugin/files/15444937/drowned_test.zip)

**Comment by MeeniMc** [comment]
You did not provide a log file so some guess work had to happen.

I tried to reproduce as best I could with 1.20.6, sodium 0.5.8, indium 1.0.30, ETF 6.0.1, EMF 2.0.2. The drowned emissives were not present, either with or without Sodium.

ETF log shows
```
[13:08:46] [Render thread/INFO] (Minecraft) Reloading ResourceManager: vanilla, fabric, ... , $polymer-resources, file/drowned_test
...
[13:08:58] [Render thread/INFO] (Entity Texture Features) Initializing texture for the first time: minecraft:textures/entity/zombie/drowned.png.
[13:08:58] [Render thread/INFO] (Entity Texture Features)  - no variants for: [minecraft:textures/entity/zombie/drowned.png, emissive=false, blinks=false].
[13:08:58] [Render thread/INFO] (Entity Texture Features) Amount of 'base' textures: 86.
[13:08:58] [Render thread/INFO] (Entity Texture Features) Total textures including variants: 88.
[13:08:58] [Render thread/INFO] (Entity Texture Features) Initializing texture for the first time: minecraft:textures/entity/trident.png.
[13:08:58] [Render thread/INFO] (Entity Texture Features)  - no variants for: [minecraft:textures/entity/trident.png, emissive=false, blinks=false].
[13:08:58] [Render thread/INFO] (Entity Texture Features) Amount of 'base' textures: 87.
[13:08:58] [Render thread/INFO] (Entity Texture Features) Total textures including variants: 89.
[13:08:58] [Render thread/INFO] (Entity Texture Features) Initializing texture for the first time: minecraft:textures/entity/zombie/drowned_outer_layer.png.
[13:08:58] [Render thread/INFO] (Entity Texture Features)  - no variants for: [minecraft:textures/entity/zombie/drowned_outer_layer.png, emissive=false, blinks=false]
```

Looks like the moremmce file has not been accounted for, that would be an ETF problem, or a problem with the pack itself.

**Comment by VindSole** [comment]
disabled ETF and emissives still don't work so this mod is not the issue, tested pack and it works without embeddium 0.3.19/sodium 0.5.8 present, and it doesn't when it is.

no sodium
![image](https://github.com/MoreMcmeta/emissive-plugin/assets/63668266/fe05fc54-9766-4eab-8905-abf5400aa3e9)

sodium enabled
![image](https://github.com/MoreMcmeta/emissive-plugin/assets/63668266/6b6ddd04-01e1-4209-bc1d-aaf8bc2cbf66)

Log when Embeddium 0.3.19 is enabled (sorry for resource pack test bloat)
[latest.log](https://github.com/user-attachments/files/15687254/latest.log)

When Sodium 0.5.8 is enabled
[latest.log](https://github.com/user-attachments/files/15687261/latest.log)

No Sodium mods
[latest.log](https://github.com/user-attachments/files/15687249/latest.log)

**Comment by MeeniMc** [comment]
Ok I thought you were using ETF to add emissives, but the critical step is using moremcmeta-emissives. This mod requires Indium to be compatible with Sodium. See their readme: https://modrinth.com/mod/moremcmeta-emissive

**Comment by VindSole** [comment]
My bad for not using Indium for logging the mod, however I have been using indium previously with the same results, moreover embeddium has native fabric API rendering support so indium is not needed.

**Comment by soir20** [comment]
The changes required to update the plugin for 1.21 have also fixed this issue for Sodium. These changes are quite substantial--it would likely take a lot of work to backport to 1.20.6 and other versions. That takes away time from me being able to investigate other mod issues. For now, I am planning to fix this in 1.21 and later versions only. I will update this issue if that changes, but I am closing it as I consider the issue fixed.

Please reopen or create a new issue if you still see this on 1.21+ versions.
